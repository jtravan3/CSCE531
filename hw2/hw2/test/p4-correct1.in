a = -25
b = +4
(-a)*b
-a-b
-a+(-b)
a-(-b)*7
a*(-b+6)
a*(-b)+6
(-a)/b
(b+(-a))%b
-(-(-(-(-b))))
