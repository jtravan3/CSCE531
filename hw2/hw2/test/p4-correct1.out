Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: -25
   2: 4
   3: 100
   4: 21
   5: 21
   6: 3
   7: -50
   8: 106
   9: 6
  10: 1
  11: -4
  12: 
